src/Extensions/Microsoft.Diagnostics.Monitoring.Extension.Common/Microsoft.Diagnostics.Monitoring.Extension.Common.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <IsShippingAssembly>true</IsShippingAssembly>
```

Comment:
This shouldn't be necessary since the Directory.Build.props sets IsShipping to true.